tests/issues/issue-2124.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+return [
+    'issue-2124-A' => [
+        'ua' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15',
+        'properties' => [
+            'Comment' => 'Safari 13.0',
+            'Browser' => 'Safari',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Apple Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '13.0',
+            'Platform' => 'macOS',
+            'Platform_Version' => '10.15',
+            'Platform_Description' => 'macOS',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Apple Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => true,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Macintosh',
+            'Device_Maker' => 'Apple Inc',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Macintosh',
+            'Device_Brand_Name' => 'Apple',
+            'RenderingEngine_Name' => 'WebKit',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Apple Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2124-A (lite)' => [
+        'ua' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15',
+        'properties' => [
+            'Comment' => 'Safari 13.0',
+            'Browser' => 'Safari',
+            'Version' => '13.0',
+            'Platform' => 'MacOSX',
+            'Device_Type' => 'Desktop',
+        ],
+        'lite' => true,
+        'standard' => false,
+        'full' => false,
+    ],
+    'issue-2124-B' => [
+        'ua' => 'Mozilla/5.0 (iPad; CPU OS 13_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Mobile/15E148 Safari/604.1',
+        'properties' => [
+            'Comment' => 'Mobile Safari 13.0',
+            'Browser' => 'Safari',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Apple Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '13.0',
+            'Platform' => 'iOS',
```

Comment:
Why is this one `iOS` in Standard/Full but `ipadOS` on Lite?

Reply:
fixed